azure/init.sh

```diff
@@ -43,6 +45,17 @@ rsync -aXS /tmp/home_copy/. /home/${TARGET_USER}/.
 # We do not want the password prompt, because we want to run tests without any user input
 echo '${TARGET_USER}     ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
 
+# we will use port 3456 to not hit security rule 103
+echo 'Port 3456' >> /etc/ssh/sshd_config
+echo 'Port 22' >> /etc/ssh/sshd_config
```

Comment:
why putting port 22 ?

Reply:
I guess it's because now we override ssh ports for custom port and so the default port would go away if we don't add it explicitly

Reply:
I kept the default port because Azure has rules based on rule 22 and it tries to connect on 22. So I thought it could also cause some problems and possibly creation of ICM if rule 22 is not working for inner communication in azure.